src/CMakeLists.txt

```diff
@@ -338,8 +338,8 @@ find_package(Qt5 5.6 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools
 # This is not present before Qt 5.7:
 find_package(Qt5 COMPONENTS Gamepad QUIET)
 
-if (Qt5Core_VERSION VERSION_LESS 5.6)
```

Comment:
I think the error messages are not required - IIRC the **main** `find_package` for the Qt libraries (the one I've already commented about) will error out or, at least protest loudly, because it is not called with the `QUiET` option!  It may however still be possible to adjust the version numbers tested and instead point out about the minimum Qt version value for the Text-To-Speech facilities - though they haven't yet been pulled-in have they?

Reply:
Just looking to raise the Qt version so I won't mess with the other unrelated stuff for now. TTS stuff is still an open PR, yep.